virtual_ecosystem/models/animal/animal_cohorts.py

```diff
@@ -697,34 +723,6 @@ def is_below_mass_threshold(self, mass_threshold: float) -> bool:
             self.mass_current + self.reproductive_mass
         ) / self.functional_group.adult_mass < mass_threshold
 
-    def inflict_natural_mortality(
```

Comment:
This function is removed but not added anywhere else and, also, not related with the purpose of the PR. Is it an error or am I missing anything?

Reply:
Both this function and metabolic_rate_energy are no longer in use. They were replaced with inflict_non_predation_mortality and metabolic_rate respectively. I forgot to remove them in the previous relevant PR. I'll try and include that in the PR description if it happens again in the future.